flint.ui/src/store/modules/point.js

```diff
@@ -175,38 +177,16 @@ export default {
       let pool_3_valuex = '#$' + pool_3_value + '#$'
       state.Point_config['Pools'][2]['Pool 3'] = pool_3_valuex
       console.log(state.Point_config)
+    },
+    save_results(state, response) {
+      state.results= response
+      console.log('results sent to state')
+      //console.log(state.results)
     }
   },
 
   actions: {
-    // send_pointConfig() {
-    //   let FLINT_config_string = JSON.stringify(
-    //     this.state.flint_point.Point_config
-    //   )
-    //   let preprocessed_FLINT_config_string = FLINT_config_string.replaceAll(
-    //     '"#$',
-    //     ' '
-    //   )
-    //   let final_FLINT_config_string = preprocessed_FLINT_config_string.replaceAll(
-    //     '#$"',
-    //     ' '
-    //   )
-    //   axios
-    //     .post('http://127.0.0.1:8080/point', final_FLINT_config_string)
-    //     .then(response => {
-    //       this._vm.$toast.success(`${response}`, { timeout: 2000 })
-    //       console.log(response)
-    //     })
-    //     .catch(error => {
-    //       this._vm.$toast.error(`${error}`, { timeout: 2000 })
-    //       console.log(error)
-    //     })
-    // },
-    set_received_point_example_data(state, response) {
-      state.received_data = response
-      console.log('received_data sent to state')
-    },
-    send_pointConfig({ commit }) {
+   send_pointConfig({ commit }) {
       let FLINT_config_string = JSON.stringify(this.state.point.Point_config)
       let preprocessed_FLINT_config_string = FLINT_config_string.replaceAll(
```

Comment:
`replaceAll` takes regular expressions so I think you can do both with `replaceAll('"#$|#$"')`.

* `preprocessed_FLINT_config_string` -> `parsed_config_string`

Reply:
@aornugent Sadly the regular expression `replaceAll('"#$|$#"')` didn't work. 😓